### PR TITLE
feat: Threshold before showing update notification

### PR DESCRIPTION
**Pull request by GeckoEidechse**
This way we don't spam the end-user with an update notification before an update is fully built and released.

### CODE REVIEW

**Comment by GeckoEidechse** [comment]
Wtf? It ran locally just fine :(

File: src-tauri/src/github/release_notes.rs [hunk 1]
```diff
@@ -52,7 +52,38 @@ pub async fn check_is_flightcore_outdated() -> Result<bool, String> {
...
+    // If outdated, check how new the update is
+    if is_outdated {
+        // Extract release date from JSON
+        let release_date = json_response
+            .get("published_at")
+            .and_then(|value| value.as_str())
+            .unwrap();
+
+        // Time to wait (1 day)  h *  m *  s
+        let threshold_seconds = 24 * 60 * 60;
```
Comment (Alystrasz) [inline src-tauri/src/github/release_notes.rs:66]: So if I get it right, users have to wait an entire day from the moment a new release is published?

Comment (GeckoEidechse) [reply src-tauri/src/github/release_notes.rs:66]: Yup

Comment (GeckoEidechse) [reply src-tauri/src/github/release_notes.rs]: Adjusted to 2 hours now with 8f255f341f4e067f98dae85dd78006c2fe8df86b

**Review by Alystrasz — APPROVED** [review]

**Pull request merged**